InvenTree/label/tests.py

```diff
@@ -154,8 +154,6 @@ def test_print_part_label(self):
         self.assertIn(f"part: {part_pk} - {part_name}", content)
         self.assertIn(f'data: {{"part": {part_pk}}}', content)
         self.assertIn(f'http://testserver/part/{part_pk}/', content)
-        self.assertIn("img/blank_image.png", content)
-        self.assertIn("img/inventree.png", content)
```

Comment:
Question: Why are those tests removed?

Reply:
Replacing these now, they are outdated with the new approach